tests/cache/jobQueue/getHierarchy.js

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/cache/jobQueue/getHierarchy.js
+ */
+'use strict';
+const jobQueue = require('../../../jobQueue/setup').jobQueue;
+const jobType = require('../../../jobQueue/setup').jobType;
+const expect = require('chai').expect;
+const supertest = require('supertest');
+const api = supertest(require('../../../index').app);
+const samstoinit = require('../../../cache/sampleStoreInit');
+const tu = require('../../testUtils');
+const rtu = require('../models/redisTestUtil');
+const constants = require('../../../api/v1/constants');
+const Subject = tu.db.Subject;
+const path = '/v1/subjects/{key}/hierarchy';
+const logger = require('../../../utils/activityLog').logger;
+
+describe(`api: GET using worker process ${path}`, () => {
+
+  before(() => {
+    tu.toggleOverride('enableWorkerProcess', true);
+    tu.toggleOverride('enqueueHierarchy', true);
+    require('../../../worker/jobProcessor');
```

Comment:
I think requiring this will call all the worker process. Can you just call the bulkUpsert worker here by doing (jobQueue.process(jobType.BULKUPSERTSAMPLES, bulkUpsertSamplesJob) and probably do the same in the other tests too?